docs/content/querying/topnmetricspec.md

```diff
@@ -28,39 +28,26 @@ The metric field can also be given as a JSON object. The grammar for dimension v
 |type|this indicates a numeric sort|yes|
 |metric|the actual metric field in which results will be sorted by|yes|
 
-## Lexicographic TopNMetricSpec
+## Dimension TopNMetricSpec
```

Comment:
I have to say that the name `TopNMetricSpec` is kind of ridiculous at this point.  It was always a little silly but now it is a step worse. it should really be `TopNSortSpec` or similar.

so instead of writing `"metric": { "type": "dimension", ... }` you could write `"sort": { "type": "dimension", ... }`

Maybe this should not be changed as part of this PR. Maybe this will never be changed.